doc/examples/transform/plot_swirl.py

```diff
@@ -56,7 +56,7 @@ def shift_left(xy):
 
     s = \mathtt{strength}
 
-    \\theta' = \phi + s \, e^{-\\rho / r + \\theta}
+    \\theta' = \phi + s \, e^{-\\rho / r} + \\theta
 
 where ``strength`` is a parameter for the amount of swirl, ``radius`` indicates
 the swirl extent in pixels, and ``rotation`` adds a rotation angle.  The
```

Comment:
```suggestion
where ``radius`` indicates the swirl extent in pixels, ``rotation`` adds a
rotation angle, and ``strength`` is a parameter for the amount of swirl. The
```
Here, I'm reordering the variable descriptions to match the order of appearance.